documentation/service_admin/proc-deploying-auth-services.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+
+include::common/attributes.adoc[]
+
+:cmdcli: oc
+
+[id='deploying-auth-services_{context}']
+= Deploying authentication services
+
+{ProductName} requires at least one authentication service to be deployed. The authentication service
+can either be none (allow all), standard (keycloak), or external (not managed by {ProductName}).
```

Comment:
none, standard, and external are all literals.

Reply:
keycloak is a product name - capitalize the K

Reply:
Fixed